client/src/chat.ts

```diff
@@ -220,8 +219,12 @@ function send(room: string, element: JQuery<HTMLElement>) {
   // Clear the chat box
   element.val("");
 
+  sendText(room, msg);
+}
+
+export function sendText(room: string, _msg: string): void {
```

Comment:
get rid of the underscore

Reply:
Lint complains about reassigning parameters. Prefer to rename to msgConst or something?

Reply:
if you have to reassign a parameter, then make a new variable that has a name describing how it was modified

Reply:
I dislike `msgWithEmojis`.

Reply:
maybe formattedMsg?

Reply:
Sure, but the only format changed is that last emoji... it really is `msg` so I think `msgRaw` is smart naming

Reply:
I guess it's less pretty as an export

Reply:
i dont feel strongly either way